app/assets/javascripts/student_profile/profile_details.jsx

```diff
@@ -249,8 +249,29 @@
       );
     },
 
+    renderIepDownloadLinks: function () {
+      const iepDocuments = this.props.iepDocuments;
+
+      return iepDocuments.map((iepDocument) => {
```

Comment:
If you like, this can be: `return iepDocuments.map(this.renderIepDownloadLink, this)`;